src/main/java/com/springboot/star_wars_character_popularity/app/controller/CharacterListDBController.java

```diff
@@ -76,6 +73,12 @@ public ModelAndView delete(@RequestParam("characterId") long id){
         characterService.deleteCharacter(id);
 
         // redirect to /api/character/list
-        return new ModelAndView("redirect:/api/characterlistdb/list");
+        return new ModelAndView("redirect:/api/characterlistdb/");
     }
-}
+
+    @GetMapping("/fetch-characters")
```

Comment:
good practice to use "/admin/fetch-characters" path, because only admin can do that. (do it in new controller)